shared/actions/favorite.js

```diff
@@ -161,6 +194,30 @@ export function favoriteList (): (dispatch: Dispatch) => void {
   }
 }
 
-export function ignoreFolder (path: string) {
-  return () => console.log('TODO: implement ignore folder action')
+export function ignoreFolder (path: string): (dispatch: Dispatch) => void {
+  return (dispatch, getState) => {
+    const folder = folderFromPath(path)
+    if (!folder) {
+      const action: FavoriteIgnore = {type: Constants.favoriteIgnore, payload: {error: true}}
+      dispatch(action)
+      return
+    }
+
+    const params : favoriteFavoriteIgnoreRpc = {
+      param: {folder},
+      incomingCallMap: {},
+      method: 'favorite.favoriteIgnore',
+      callback: error => {
+        if (error) {
+          console.warn('Err in favorite.favoriteIgnore', error)
+          return
```

Comment:
If there is an error, are you stuck somewhere? since you aren't navigating back?